packages/otel/src/instrumentations/vercel-handler.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  TracerProvider,
+  MeterProvider,
+  propagation,
+  context,
+} from "@opentelemetry/api";
+import type {
+  Instrumentation,
+  InstrumentationConfig,
+} from "@opentelemetry/instrumentation";
+import { Server } from "node:http";
+
+export interface VercelHandlerInstrumentationConfig
+  extends InstrumentationConfig {}
+
+export class VercelHandlerInstrumentation implements Instrumentation {
+  instrumentationName = "@vercel/otel/handlers";
+  instrumentationVersion = "1.0.0";
+  instrumentationDescription?: string;
+
+  traceProvider?: TracerProvider;
```

Comment:
Typo i think (trace**r**Provider):
```suggestion
  tracerProvider?: TracerProvider;
```